### PR TITLE
upgrade sdk

### PR DESCRIPTION
**Pull request by jhernandezb**

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -118,35 +122,24 @@ func (p *Proxy) SubscribeNewBlocks(subscriber string) (<-chan tmctypes.ResultEve
...
-	if err := p.cdc.UnmarshalJSON(bz, &tx); err != nil {
-		return sdk.TxResponse{}, err
-	}
-	return tx, nil
+func (p *Proxy) Tx(hash string) (*sdk.TxResponse, error) {
+	initClientCtx := client.Context{}.
+		WithJSONMarshaler(p.encodingConfig.Marshaler).
+		WithInterfaceRegistry(p.encodingConfig.InterfaceRegistry).
+		WithTxConfig(p.encodingConfig.TxConfig).
+		WithLegacyAmino(p.encodingConfig.Amino).
```
Comment (shanev) [inline client/client.go:130]: So this is still using Amino?

**Comment by shanev** [comment]
I'd also check if there's been any updates to Juno and merge with upstream.

File: client/client.go (hunk 1 above)
Comment (jhernandezb) [reply client/client.go:130]: Its mostly just wiring, don't think we really need it but leaving it for now.

**Comment by jhernandezb** [comment]
> 
> 
> I'd also check if there's been any updates to Juno and merge with upstream.

I believe juno is discontinued, not really new updates since february.

**Pull request merged**